src/Algorithms/TMJets/TMJets20/reach.jl

```diff
@@ -211,19 +211,19 @@ function validated_step!(f!, t::Taylor1{T}, x::Vector{Taylor1{TaylorN{T}}},
         t0::T, tmax::T, sign_tstep::Int,
         xTMN::Vector{TaylorModelN{N,T,T}}, xv::Vector{IntervalBox{N,T}},
         rem::Vector{IA.Interval{T}}, zbox::IntervalBox{N,T}, symIbox::IntervalBox{N,T},
-        nsteps::Int, orderT::Int, abstol::T, params, parse_eqs::Bool,
+        nsteps::Int, orderT::Int, abstol::T, params, tmpTaylor, arrTaylor, parse_eqs::Bool,
         check_property::Function=(t, x)->true, adaptive::Bool=true) where {N,T}
 
     # One step integration (non-validated)
     # TaylorIntegration.__jetcoeffs!(Val(parse_eqs), f!, t, x, dx, xaux, params)
     # δt = TaylorIntegration.stepsize(x, abstol)
-    δt = TaylorIntegration.taylorstep!(f!, t, x, dx, xaux, abstol, params, parse_eqs)
+    δt = TaylorIntegration.taylorstep!(f!, t, x, dx, xaux, abstol, params, tmpTaylor, arrTaylor, parse_eqs)
     f!(dx, x, params, t)  # Update `dx[:][orderT]`
 
     # One step integration for the initial box
     # TaylorIntegration.__jetcoeffs!(Val(parse_eqs), f!, tI, xI, dxI, xauxI, params)
     # δtI = TaylorIntegration.stepsize(xI, abstol)
-    δtI = TaylorIntegration.taylorstep!(f!, tI, xI, dxI, xauxI, abstol, params, parse_eqs)
+    δtI = TaylorIntegration.taylorstep!(f!, tI, xI, dxI, xauxI, abstol, params, tmpTaylor, arrTaylor, parse_eqs)
```

Comment:
@lbenet is it okay to pass the same `tmpTaylor, arrTaylor` in the second call? [Here](https://github.com/JuliaIntervals/TaylorModels.jl/pull/133/files#diff-da99092bea9a3cd084ea4ff4799b42bddc499391d11ca08288ff944e12a9033bL832-R842) you pass something different.

Reply:
Good point! `tmpTaylor` and `arrTaylor` are temporal vectors and vector of vectors, respectively, with known (and fixed) types. My guess is that the second integration, which involves interval coefficients, would break that and throw some error. So I think you need different temporal arrays.

Below, around line 331, you could add the following line:
```julia
parse_eqsI, tmpTaylorI, arrTaylorI = TaylorIntegration._determine_parsing!(parse_eqs, f!, tI, xI, dxI, params)
```
which should create the temporal arrays needed for the second integration, which would be needed as parameters in `validated_step!`.

Reply:
See [line 333](https://github.com/JuliaIntervals/TaylorModels.jl/blob/259a87e406eefe59a02fdbdc4fd8805b5e77e77b/src/validatedODEs.jl#L333), [line 338](https://github.com/JuliaIntervals/TaylorModels.jl/blob/259a87e406eefe59a02fdbdc4fd8805b5e77e77b/src/validatedODEs.jl#L338) and [line 576-579](https://github.com/JuliaIntervals/TaylorModels.jl/blob/259a87e406eefe59a02fdbdc4fd8805b5e77e77b/src/validatedODEs.jl#L576) of TaylorModels.jl/src/validatedODEs.jl.

Reply:
Thanks! I added that change.